synapse/handlers/oidc_handler.py

```diff
@@ -117,7 +117,6 @@ def __init__(self, hs: "HomeServer"):
         self._http_client = hs.get_proxied_http_client()
         self._auth_handler = hs.get_auth_handler()
         self._registration_handler = hs.get_registration_handler()
-        self._server_name = hs.config.server_name  # type: str
```

Comment:
Honestly I'd rather we went in the opposite direction: removing the stuff that relies on BaseHandler, and removing the inheritance from it. But :man_shrugging:

Reply:
I don't have a strong opinion on this I can back out the change.